heartbeat/podman-etcd

```diff
@@ -611,7 +633,7 @@ add_member_as_learner()
 	local member_ip=$2
 
 	ocf_log info "add $member_name ($member_ip) to the member list as learner"
-	out=$(podman exec "${CONTAINER}" etcdctl --endpoints="https://$(attribute_node_ip get):2379" member add "$member_name" --peer-urls="https://$member_ip:2380" --learner)
+	out=$(podman exec "${CONTAINER}" etcdctl --endpoints="$(ip_url $(attribute_node_ip get)):2379" member add "$member_name" --peer-urls="$(ip_url $member_ip):2380" --learner)
```

Comment:
For better clarity, let's declare the `endpoint_url` and `peer_url` variables first.